src/repl.ts

```diff
@@ -214,13 +215,53 @@ function _eval(service: Service, state: EvalState, input: string) {
   const undo = appendEval(state, input);
   let output: string;
 
+  function adjustUseStrict(code: string) {
+    // "void 0" keeps the repl from returning "use strict" as the result
+    // value for statements and declarations that don't return a value.
+    return code.replace(/^"use strict";/, '"use strict"; void 0;');
```

Comment:
Is this a separate enhancement, unrelated to top-level await?  If so, I will create a separate issue to track it.  We can still merge it as part of this pull request, but it will be nice to list it separately and give you credit in our release notes.

Reply:
Yes correct, if you want I can create a separate PR that deals with just that.

Reply:
It's your decision.  I think it will be easier and faster to keep the change in this PR, so I'm happy to keep it here.

I see it is causing test failures where our tests are written expecting the repl to output `'use strict'` instead of `undefined`.  We can update those tests within this PR as well.

I've created #1387 to describe the bug.

Reply:
I've made #1388 for fixing this, so this PR remains relevant to only one thing.